examples/jss/src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { render } from 'react-dom';
 
-import './global-styles';
+import GlobalStyles from './components/global-styles';
 import App from './components/app';
 
-render(<App />, document.getElementById('example-root'));
+render(<GlobalStyles><App /></GlobalStyles>, document.getElementById('example-root'));
```

Comment:
Because in jss, selectors are namespaced by default. It can not now if you mean html as a tag selector or html as a name for your rule. In order to differentiate, jss has 2 rendering mechanisms. One is to put named: false to the sheet, this makes use all property names as selectors. Another is to use nested selectors.